utils/tweet_query.py

```diff
@@ -29,7 +30,7 @@ def __init__(self,
 
 	def pull(self, query:str, output_dir: str, 
 				start_time: Union[datetime, str] = None, end_time: Union[datetime, str] = None,
-				max_results: int = 100): # add start and end times
+				max_results: int = 100, batch_size: int = 100): # add start and end times
```

Comment:
Why are there separate batch size and max results? Aren't these the same thing?

Reply:
batch_size gets passed to the Tweepy call as the number of tweets to fetch in a single request to the API.  max_results is the total number of tweets to fetch after all calls to the API, i.e. the total number of tweets after all batches are fetched.